_blueprints/developers_guide.adoc

```diff
@@ -0,0 +1,17 @@
+= Developer's Guide
+
+== Adding new packages
+
+ALMighty uses https://glide.sh[Glide] for package management.  To
+install a new package, you can follow these steps.  The `<URL>` should
+be replaced with the package URL pointing to Github or any other
```

Comment:
Yes, it's not a proper URL, though it is mentioned as a URL in the official docs when talking about remote packages (https://golang.org/doc/code.html#remote) The most appropriate word is **import path** and package name is just the last part of the import path.